references/segmentation/train.py

```diff
@@ -134,8 +135,8 @@ def main(args):
     else:
         torch.backends.cudnn.benchmark = True
 
-    dataset, num_classes = get_dataset(args.data_path, args.dataset, "train", get_transform(True, args))
-    dataset_test, _ = get_dataset(args.data_path, args.dataset, "val", get_transform(False, args))
+    dataset, num_classes = get_dataset(args.data_path, args.dataset, "train", get_transform(train=True, args=args))
+    dataset_test, _ = get_dataset(args.data_path, args.dataset, "val", get_transform(train=False, args=args))
```

Comment:
This has nothing to do with the new transforms, but I despise calling functions with positional bool arguments if it isn't abundantly what it means. That is not the case here. Using keyword arguments makes this much easier to understand.

Reply:
Sounds good.